pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/DataTest.java

```diff
@@ -54,7 +67,20 @@ void unsignedByteTest(Integer value) throws IOException {
     }
 
     static Stream<Integer> intsTestCases() {
-        return Stream.of(Integer.MIN_VALUE,-100,-66,-7,-1,0,1,9,51,101,Integer.MAX_VALUE).map(Number::intValue);
+        return Stream.of(
```

Comment:
Oh noes, some of my unit tests also touch this, lol.

Reply:
Whoever merges first will avoid resolving conflicts )